### PR TITLE
Lr scheduler notebook

### PR DESCRIPTION
**Pull request by SamSandwich07**
Adds an notebook demonstrating the use of an LR scheduler

### CODE REVIEW

**Review by drewoldag — CHANGES_REQUESTED** [review]

File: docs/pre_executed/using_lr_schedulers.ipynb [hunk 1]
```diff
@@ -0,0 +1,400 @@
+{
```
Comment (drewoldag) [inline docs/pre_executed/using_lr_schedulers.ipynb]: <p>In the same way that you called out that the model performance was worse with gamma=0.9, you could do so here. Noting that the goal is to demonstrate the use of different learning rate schedulers more than to tune the model for optimal performance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/hyrax/pull/702/discussion/#comment-2830778686'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by drewoldag — APPROVED** [review]
This is looking very good. Thanks for the effort!

**Review by drewoldag — CHANGES_REQUESTED** [review]
The second image name changed.

**Review by drewoldag — APPROVED** [review]
Good to go
